miso-web/src/main/webapp/scripts/list_dilution.js

```diff
@@ -47,30 +115,43 @@ ListTarget.dilution = {
         ListUtils.idHyperlinkColumn("Library Name", "library", "library.id",
             function(dilution) {
               return dilution.library.name;
-            }, 0),
+            }, 0, !config.library),
         ListUtils.labelHyperlinkColumn("Library Alias", "library", function(
             dilution) {
           return dilution.library.id;
-        }, "library.alias", 0), {
+        }, "library.alias", 0, !config.library),
+        {
           "sTitle" : "Creator",
           "mData" : "dilutionUserName",
           "include" : true,
           "iSortPriority" : 0
-        }, {
+        },
+        {
           "sTitle" : "Creation Date",
           "mData" : "creationDate",
           "include" : true,
           "iSortPriority" : 0
-        }, {
+        },
+        {
           "sTitle" : "Platform",
           "mData" : "library.platformType",
           "include" : true,
           "iSortPriority" : 0
-        }, {
+        },
+        {
           "sTitle" : "Concentration",
           "mData" : "concentration",
           "include" : true,
           "iSortPriority" : 0
+        },
+        {
+          "sTitle" : "Targeted Sequencing",
+          "mData" : "targetedSequencingId",
+          "include" : Constants.isDetailedSample,
+          "mRender" : ListUtils.render.textFromId(
+              Constants.targetedSequencings, 'alias', '(None)'),
```

Comment:
Something about this isn't working -- searching `LDI644` indicates that this dilution does not have a TS value, but clicking through shows that there is one associated.